src/posthog-surveys.ts

```diff
@@ -51,12 +49,10 @@ function getRatingBucketForResponseValue(responseValue: number, scale: number) {
 }
 
 export class PostHogSurveys {
-    instance: PostHog
     private _decideServerResponse?: boolean
-    public _surveyEventReceiver: SurveyEventReceiver | null
+    public _surveyEventReceiver: IReceiveSurveyEvents | null
```

Comment:
this is one bit of the size saving, we don't have a concrete reference for this thing until we load the survey script and can grab the class from it